frontend/src/App.svelte

```diff
@@ -50,6 +71,8 @@
       window.clearTimeout(showingTimeout);
       showing = 0;
       showMore();
+      genEdQuery = (query.toLowerCase().indexOf("gened") !== -1);
+      genEdChecks = genEdQuery ? genEdChecks : genEdChecks.fill(false); 
```

Comment:
This should not be in the `data.subscribe` hook as that is run _after_ the request is made to the server and returned. It should be a computed variable instead. For example, at declaration:

```ts
$: genedQuery = query.toLowerCase().includes("gened");
$: if (!genedQuery) {
  genedChecks.fill(false);
}
```

Reply:
That makes sense—thanks for the note!